### PR TITLE
fix ocis config parsing for subcommands

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
fixes config parsing for subcommands

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Web-Tests-ocis-smoke-ocis-storage-2</strong>](https://drone.owncloud.com/owncloud/ocis/11210/35/1) failed. Further test are cancelled...

File: ocis/pkg/command/storageappprovider.go [hunk 1]
```diff
@@ -13,9 +15,13 @@ func StorageAppProviderCommand(cfg *config.Config) *cli.Command {
 		Name:     "storage-app-provider",
 		Usage:    "start storage app-provider service",
 		Category: "extensions",
-		Action: func(c *cli.Context) error {
-			origCmd := command.AppProvider(cfg.AppProvider)
-			return handleOriginalAction(c, origCmd)
+		Before: func(c *cli.Context) error {
```
Comment (butonic) [inline ocis/pkg/command/storageappprovider.go:18]: Action went missing!

**Review by butonic — CHANGES_REQUESTED** [review]
https://github.com/owncloud/ocis/pull/3642/files#r862748342

File: ocis/pkg/command/storageappprovider.go (hunk 1 above)
Comment (wkloucek) [reply ocis/pkg/command/storageappprovider.go:19]: I brought it back

**Review by dragonchaser — APPROVED** [review]
lgtm

**Pull request merged**